post_installation_configuration/preparing-for-users.adoc

```diff
@@ -0,0 +1,145 @@
+|Configure a `github` identity provider to validate user names and passwords
+against GitHub or GitHub Enterprise's OAuth authentication server.
+
+|xref:../authentication/identity_providers/configuring-gitlab-identity-provider.adoc#configuring-gitlab-identity-provider[GitLab]
+|Configure a `gitlab` identity provider to use
+link:https://gitlab.com/[GitLab.com] or any other GitLab instance as an identity
+provider.
+
+|xref:../authentication/identity_providers/configuring-google-identity-provider.adoc#configuring-google-identity-provider[Google]
+|Configure a `google` identity provider using
+link:https://developers.google.com/identity/protocols/OpenIDConnect[Google's OpenID Connect integration].
+
+|xref:../authentication/identity_providers/configuring-oidc-identity-provider.adoc#configuring-oidc-identity-provider[OpenID Connect]
+|Configure an `oidc` identity provider to integrate with an OpenID Connect
+identity provider using an
+link:http://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth[Authorization Code Flow].
+
+|===
+
+Once an identity provider has been defined, you can
+xref:../authentication/using-rbac.adoc#authorization-overview_using-rbac[use RBAC to define and apply permissions].
+
+include::modules/identity-provider-parameters.adoc[leveloffset=+1]
+
+include::modules/identity-provider-default-CR.adoc[leveloffset=+1]
+
+[id="post-install-using-rbac-to-define-and-apply-permissions"]
+== Using RBAC to define and apply permissions
+
+include::modules/rbac-overview.adoc[leveloffset=+1]
+
+include::modules/rbac-projects-namespaces.adoc[leveloffset=+1]
+
+include::modules/rbac-default-projects.adoc[leveloffset=+1]
+
+include::modules/rbac-viewing-cluster-roles.adoc[leveloffset=+1]
+
+include::modules/rbac-viewing-local-roles.adoc[leveloffset=+1]
+
+include::modules/rbac-adding-roles.adoc[leveloffset=+1]
+
+include::modules/rbac-creating-local-role.adoc[leveloffset=+1]
+
+ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
+include::modules/rbac-creating-cluster-role.adoc[leveloffset=+1]
+endif::[]
+
+include::modules/rbac-local-role-binding-commands.adoc[leveloffset=+1]
+
+ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
+include::modules/rbac-cluster-role-binding-commands.adoc[leveloffset=+1]
+
+include::modules/rbac-creating-cluster-admin.adoc[leveloffset=+1]
+endif::[]
+
+include::modules/authentication-kubeadmin.adoc[leveloffset=+1]
+
+include::modules/authentication-remove-kubeadmin.adoc[leveloffset=+1]
+
+[id="post-install-image-configuration-resources"]
+== Image configuration resources
+
+include::modules/images-configuration-parameters.adoc[leveloffset=+1]
+
+include::modules/images-configuration-file.adoc[leveloffset=+1]
+
+include::modules/images-configuration-cas.adoc[leveloffset=+2]
+
+include::modules/images-configuration-insecure.adoc[leveloffset=+2]
+
+include::modules/images-configuration-registry-mirror.adoc[leveloffset=+2]
```

Comment:
Issues in images-configuration-registry-mirror.adoc
1.   `/etc/containers/registries` should be ` /etc/containers/registries.conf`
2. In `Procedure
Configure mirrored repositories. To do that, you can either:` field , the cmd `skopeo copy \
   docker://registry.access.redhat.com/ubi8/ubi-minimal@sha256:c505667389712dc337986e29ffcb65116879ef27629dc3ce6e1b17727c06e78f \
   docker://example.io/ubi8/ubi-minimal` should be `skopeo copy \
   docker://registry.access.redhat.com/ubi8/ubi-minimal@sha256:c505667389712dc337986e29ffcb65116879ef27629dc3ce6e1b17727c06e78f \
   docker://example.io/example/ubi-minimal`